### DISCUSSION TITLE
understanding conda-store failures

### DISCUSSION THREAD

**Discussion by benjaminleighton** [nebari-dev/nebari#936] [Q&A]
Hi, I'm finding it difficult to debug failing conda enivironment specifications is there a recommended pattern? At the moment through logs on the conda-store i can see and ERROR reporting the environment failed to build but no other information.

**Comment by pierrotsmnrd** [comment]
Hi @benjaminleighton, thanks for opening this discussion.

Which version of QHub do you use ? 
Also, is there any relevant or useful information here ? https://docs.qhub.dev/en/stable/source/admin_guide/troubleshooting.html 

I'm pinging @costrouc  the architect of conda-store, also involved in QHub, who may bring a more direct answer :)

**Reply by benjaminleighton** [reply]
Thanks @pierrotsmnrd for getting back to me so quickly. I'm using 0.3.13 deploying to aws https://docs.qhub.dev/en/stable/source/admin_guide/troubleshooting.html is useful in that it points me to k9s for debugging, I haven't tried it yet but will do. At the moment I'm using the vscode kubernetes plugin to interrogate the cluster and the logs on the conda-store aren't very informative by default apart from indicating an error building the environment.

**Reply by pierrotsmnrd** [reply]
Ok ! thanks for giving us more details. Let us know if the k9s debugging helps you solve this and access more detailed logs. 

Another question : does your environment get solved using directly conda on another machine ? If not, it may mean that your environment spec is unsolvable, leading to the error. More details and logs are indeed necessary though.

**Reply by benjaminleighton** [reply]
Yes I think the problem is my conda-environment is unsovable. I haven't tried solving it elsewhere. I was hoping I could conda-store in qhub would directly report issues with my configuration solvability.  I should give it a go elsewhere though. Thanks

**Reply by pierrotsmnrd** [reply]
Ok, let us know if that's the reason :) In any case, we're taking note that having access to logs in a more simple, direct way, would make life easier for our users.

**Reply by dharhas** [reply]
So one of the problems in the current version of conda-store deployed on QHub is that getting the logs is difficult. That should be fixed in the next release. There are also several other improvements that will be in that release. 

There is also a new command that retrievers logs but I think you need to be on the next release for that as well.

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
